attribute/genericattribute.go

```diff
@@ -11,8 +11,10 @@ import (
 //GenericAttribute is a Yoti attribute which returns a generic value
 type GenericAttribute struct {
 	*yotiprotoattr_v3.Attribute
-	Value   interface{}
-	Anchors []*anchor.Anchor
+	value     interface{}
+	anchors   []*anchor.Anchor
```

Comment:
Does this hold a list of Anchors e.g [AnchorObj1, AnchorObj2, ...] or a Map of Anchors as ['1.3.6.1.4.1.47127.1.1.1' => AnchorObj, ...]?

Reply:
Yes this is a slice, a bit like a list - there's basically nothing a list can do that a slice can't (in Go), but lists have the disadvantage of not being strongly typed. You can have a [map](https://blog.golang.org/go-maps-in-action), but I don't think that is necessary here